src/main/java/uk/ac/ebi/embl/api/validation/helper/MasterSourceFeatureUtils.java

```diff
@@ -138,7 +147,11 @@ public SourceFeature constructSourceFeature(SampleEntity sample, TaxonHelper tax
 		}
 
 		if (latitude != null && longitude != null) {
-			addSourceQualifier(Qualifier.LAT_LON_QUALIFIER_NAME, latitude + " " + longitude, sourceFeature);
+			String latDirect = latitude < 0 ? "S" : "N";
```

Comment:
what if latitude and longitude value already has direction indicator (S,N or something)